tools/wpt/browser.py

```diff
@@ -27,6 +28,14 @@ def _get_fileversion(binary, logger=None):
         return None
 
 
+def handleRemoveReadonly(func, path, exc):
```

Comment:
Snake case is preferred:
```suggestion
def handle_remove_readonly(func, path, exc):
```

Reply:
Thanks, this should be fixed now.